File: java-main/java-main/JavaDSA_Super-X/Assi_1/p2.java
import java.util.Scanner;

class Demo{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter the number of rows: ");
        int rows = input.nextInt();
        input.close();

        int num = 1;

        for (int i = 0; i < rows; i++) {
            int x = num;
            for (int j = 0; j <= i; j++) {
                System.out.print(x + " ");
                x++;
            }
            num += i + 1;
            System.out.println();
        }
    }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session4/p2.c
/* 1 2 3 4 
   1 2 3
   1 2
   1
   */

#include<stdio.h>
void main(){

	int rows;

	printf("Enter no of rows:\n");
	scanf("%d",&rows);

	for(int i=1;i<=rows;i++){
		int num=1;
		for(int j=rows;j>=i;j--){
			printf("%d ",num);
			num++;
		}
		printf("\n");
	}
}

 
